frontend/src/components/common/ProfileSidebar.vue

```diff
@@ -119,8 +119,11 @@ const buttonLabel = computed(() => {
     display: flex;
     margin: 1.875rem 0 0;
 
-    .img-wrapper {
+    .profile-avatar {
         margin-right: 2rem;
+        margin-top: 0.5rem;
+        background: $light-background-brand;
+        color: $dark-text-primary;
```

Comment:
hardcode the avatar background and text color for now, until we have a better idea of what the avatar should look like